src/ResourceManager/HDInsight/Commands.HDInsight/JobCommands/GetAzureHDInsightJobOutputCommand.cs

```diff
@@ -133,25 +134,38 @@ private static string Convert(Stream stream)
 
         internal IStorageAccess GetDefaultStorageAccess(string resourceGroupName, string clusterName)
         {
+            var StorageAccountSuffix = "";
```

Comment:
Note that DefaultContext.Environment contains the appropriate storage suffix for the current azure environment (Azure, MoonCake, Germany, Government, STack), why are you not usign this value?